### ISSUE TITLE
Add XMP crs:AutoTone tag

### ISSUE THREAD

**Issue by ncruces** [exiftool/exiftool#8]
Would it be possible to add this undocumented tag?

It shows up in standalone XMP files used as Lightroom presets (created through [this](https://feedback.photoshop.com/photoshop_family/topics/idea-auto-exposure-in-addition-to-auto-tone) screen, or similar).

I'm putting the following in my `.ExifTool_config`:
```
%Image::ExifTool::UserDefined = (
    'Image::ExifTool::XMP::crs' => {
        AutoTone  => { Writable => 'boolean' },
    },
);
```

I would do a PR, but I'm not sure I know everything that needs to be updated (documentation, tests).

Thanks!

**Comment by boardhead** [comment]
Thanks.  This will be added in ExifTool 11.29

**Comment by boardhead** [comment]
Resolved by ExifTool 11.29
